MidasCivil_Adapter/CRUD/Read/Properties/MetaData.cs

```diff
@@ -0,0 +1,144 @@
+            Review review1 = new Review();
+            Review review2 = new Review();
+            Review review3 = new Review();
+            Review review4 = new Review();
+
+            foreach (string dataItem in Data)
+            {
+
+                if (dataItem.Contains("PROJECT=")) { metaData.ProjectNumber = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("REVISION=")) { metaData.Revision = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("USER=")) { metaData.Author = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("EMAIL=")) { metaData.Email = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("ADDRESS=")) { metaData.Location = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("CLIENT=")) { metaData.Client = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("TITLE=")) { metaData.ProjectName = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains("EDATE="))
+                {
+                    metaData.CreationDate = (DateTime)convertDate(dataItem.Split('=')[1]);
+                }
+                else if (dataItem.Contains(";DESIGNSTAGE=")) { metaData.DesignStage = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains(";PROJECTLEAD=")) { metaData.ProjectLead = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains(";DESCRIPTION=")) { metaData.Description = dataItem.Split('=')[1]; }
+                else if (dataItem.Contains(";DISCIPLINE=")) { metaData.Discipline = dataItem.Split('=')[1]; }
+
+                if (dataItem.Contains("CHECK1="))
+                {
+                    review1.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review1);
+                }
+                else if (dataItem.Contains("CHECK2="))
+                {
+                    review2.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review2);
+                }
+                else if (dataItem.Contains("CHECK3="))
+                {
+                    review3.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review3);
+                }
+                else if (dataItem.Contains("APPROVE="))
+                {
+                    review4.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review4);
+                    review4.Approved = true;
+                }
+                if (dataItem.Contains("CDATE1=")) { review1.ReviewDate = (DateTime)convertDate(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("CDATE2=")) { review2.ReviewDate = (DateTime)convertDate(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("CDATE3=")) { review3.ReviewDate = (DateTime)convertDate(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("ADATE=")) { review4.ReviewDate = (DateTime)convertDate(dataItem.Split('=')[1]); }
```

Comment:
Updated captial C, DateTime? wont convert to DateTime without a prompt.

Reply:
Yes, `ConvertDate` is a method so should be capitalised - also when you put it in the `Convert` class rename the method `Date` so that it become `BH.Engine.Adapters.MidasCivil.Convert.Date()`.

You shouldn't need to cast it though, because `Date` will return a `DateTime`.